applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -819,6 +819,8 @@ attachment_name_by_media_type(CT) ->
 %%--------------------------------------------------------------------
 -spec summary(cb_context:context()) -> cb_context:context().
 summary(Context) ->
+    Account = cb_context:auth_account_id(Context),
+    erlang:put(<<"is_system_admin">>, wh_util:is_system_admin(Account)),
```

Comment:
Why not, when calling `crossbar_doc:load_view/3` in `summary_{available,account}`, set the normalize function based on `wh_util:is_system_admin(Account)`?

```erlang
NomalizeFun = select_normalize_function(wh_util:is_system_admin(Account)),
crossbar_doc:load_view(?CB_LIST, [], Context, NormalizeFun)
...

select_normalize_function('true') -> fun normalize_available/2;
select_normalize_function('false') -> fun normalize_non_admin_available/2.
```

Naming is not the greatest but that should avoid the process dictionary and be clearer in intent.